style/main.scss

```diff
@@ -76,37 +76,17 @@ $windowHeight: 400;
 
 .window {
   background: $white;
-  height: calc(100% - 40px);
 
   box-sizing: border-box;
   z-index: 1000;
+  position: absolute;
 
-  margin: 20px;
-  padding: 10px;
-  border: 6px solid $lightgray-dark;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
```

Comment:
I really thought about doing it differently. But I think it's supposed to be like that? :/